SECURITY.md

```diff
@@ -0,0 +1,116 @@
+# Security Release Process
+
+The Kyverno community has adopted this security disclosures and response policy to ensure we responsibly handle critical issues.
+
+## Reporting a Vulnerability
+
+### When you should?
+- You think you discovered a potential security vulnerability in Kyverno
```

Comment:
Just a nit before deeper review:
Please make sure to have a full stop at the end of each sentence & make sure empty lines are consistent (sometimes it's one or two empty lines between headings).

Reply:
Thanks for the suggestion. Didn't notice it. Have made the empty lines consistent and added full stops wherever necessary